Assets/Scripts/Blocks/Block.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public enum BlockType { StartBlock = 0, GoalBlock = 1, EmptyBlock = 2, WoodBlock = 3 };
```

Comment:
```suggestion
public enum BlockType { Start= 0, Goal= 1, Empty= 2, Wood= 3 };
```
Wenn das Enum schon BlockType heißt ist es Redundant die Konstanten auch noch Block zu nennen.